MatrixKit/Views/RoomBubbleList/MXKRoomBubbleTableViewCell.h

```diff
@@ -82,6 +82,14 @@ extern NSString *const kMXKRoomBubbleCellUserIdKey;
 extern NSString *const kMXKRoomBubbleCellEventKey;
 
 #pragma mark - MXKRoomBubbleTableViewCell
+
+typedef NS_ENUM(NSInteger, ReadReceiptsAlignment) {
```

Comment:
I suggest moving this enumerate in MXKReceiptAvartarsContainer.h to avoid cross-reference.